projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -367,10 +367,250 @@ export interface OccEndpoints {
    * Endpoint for place order
    */
   placeOrder?: string | OccEndpoint;
+  /**
+   * Endpoint for userGroupOrderApprovalPermission
+   *
+   * @member {string}
+   */
+  budget?: string | OccEndpoint;
```

Comment:
It feels natural to me to move OCC endpoints of MyOrg to the MyOrg library. That said, I can understand it's challenging to split an interface. 

In the end of the day it's about type safety and intellisense hinting for customers 

Can we leverage somehow the type augmentation for that @dunqan ?

Reply:
I added ticket for it https://github.com/SAP/spartacus/issues/9112